site/docs/tasks/high-availability.md

```diff
@@ -0,0 +1,9 @@
+# High Availability
+
+When having more then one manager instance running [Active/Passive high availability](https://docs.openstack.org/kuryr-kubernetes/latest/devref/high_availability.html) method is used: only one of them is being the leader (doing all the exclusive tasks) and other instances waiting in standby mode in case the leader dies to take over the leader role.
```

Comment:
Maybe better to remove this very specific product link, and add something else, if we cannot add something generic to Kubernetes, we can add something product-neutral  or do not add anything ;-)